ui/app/components/task-row-port.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+import isIp from 'is-ip';
+
+export default Component.extend({
+  tagName: 'li',
+
+  href: computed('network.ip', 'port.Value', function() {
+    const ip = this.get('network.ip');
+    const port = this.get('port.Value');
+
+    let ipHrefPortion;
+
+    if (isIp.v6(ip)) {
```

Comment:
This [adds 688 bytes](https://bundlephobia.com/result?p=is-ip@3.1.0) which seems fine to me 🤔

Reply:
Yeah, seems fine. Plus I'd rather not be responsible for this: https://github.com/sindresorhus/ip-regex/blob/master/index.js

Reply:
heh exactly, I was 😳 when I saw that